Formula/dnsviz.rb

```diff
@@ -26,21 +26,6 @@ class Dnsviz < Formula
     depends_on "swig"
   end
 
-  resource "dnspython" do
-    url "https://files.pythonhosted.org/packages/13/27/5277de856f605f3429d752a39af3588e29d10181a3aa2e2ee471d817485a/dnspython-2.1.0.zip"
-    sha256 "e4a87f0b573201a0f3727fa18a516b055fd1107e0e5477cded4a2de497df1dd4"
-  end
-
-  resource "M2Crypto" do
-    url "https://files.pythonhosted.org/packages/aa/36/9fef97358e378c1d3bd567c4e8f8ca0428a8d7e869852cef445ee6da91fd/M2Crypto-0.37.1.tar.gz"
-    sha256 "e4e42f068b78ccbf113e5d0a72ae5f480f6c3ace4940b91e4fff5598cfff6fb3"
-  end
-
-  resource "pygraphviz" do
-    url "https://files.pythonhosted.org/packages/3a/d6/2c56f09ee83dbebb62c40487e4c972135661b9984fec9b30b77fb497090c/pygraphviz-1.7.zip"
-    sha256 "a7bec6609f37cf1e64898c59f075afd659106cf9356c5f387cecaa2e0cdb2304"
-  end
-
```

Comment:
seems like these are still dependencies, not sure what went wrong

Reply:
Good catch, it got removed when running `bump-formula-pr` command, and I cannot include  extra deps as there is some issue with `M2Crypto`

Reply:
This happened before as well, #69409